### PR TITLE
completed text , list and modal

### PR DESCRIPTION
**Pull request by archit7448**
https://agitated-hawking-b2d497.netlify.app/components/list/list

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (sudipkundu999) [inline .vscode/settings.json]: Keep to mind to remove this everytime or just put this in git-ignore

File: Components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    margin: 1rem 0;
+}
+.list-item:hover{
+    color: var(--color-neon-blue);
+}
+.stacked-child{
+ display: flex;
+ align-items: center;
+ padding: 1rem;  
+}
```
Comment (sudipkundu999) [inline Components/list/list.css:19]: Maintain consistency in the spacing!
You can use prettier

Comment (archit7448) [reply Components/list/list.css:19]: yeah I will used it

**Review by DivyanshSareen — APPROVED** [review]
Looks good. Do the changes Sudip mentioned and you are good to merge 👍

**Review by sudipkundu999 — APPROVED** [review]
Good to merge with changes.
One suggestion: To not go with full black as a background for code. Use a lighter shade of black.

**Pull request merged**